notebooks/work-in-progress/explore_fuel_allocation.ipynb

```diff
@@ -2,8 +2,8 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    IDX_U = ['plant_id_eia', 'unit_id_pudl', 'report_date']
</pre><p>Can we use IDX_UNIT for clarity here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>